src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -66,17 +67,26 @@ public function __construct(LanguageManagerInterface $languageManager, ConfigFac
      *
      * @command locale:check
      * @aliases locale-check
+     * @option projects A comma-separated list of projects to update. If omitted, all projects will be updated.
      * @validate-module-enabled locale
      */
-    public function check()
+    public function check($options = ['projects' => ''])
     {
         $this->getModuleHandler()->loadInclude('locale', 'inc', 'locale.compare');
 
+        $projects = [];
+        if ($passed_projects = $options['projects']) {
+            $projects = StringUtils::csvToArray($passed_projects);
+        }
+
+        // Remove duplicate projects.
```

Comment:
I'm not fond of documentation that only describes what the code does. If it does not add value, it is an additional maintenance burden.

Reply:
basarkar, thanks for contributing this code. I agree that it is an added value to the existing command.

It also needs a test.

Reply:
@basarkar, Please explain the purpose of an selective `locale:check` command if the `local:update` command is not selective. A todo in the code of the latter, even suggests it is broken.